css/modal.css

```diff
@@ -0,0 +1,126 @@
+.modal {
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  top: 55%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  transition: 200ms ease-in-out;
+  border: 1px solid var(--white-color);
+  border-radius: 5px;
+  z-index: 5;
+  background-color: var(--primary-color);
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  width: 80%;
+  padding: 0 1.5rem;
+}
+
+.modal.active {
+  transform: translate(-50%, -50%) scale(1);
+}
+
+.modal-header {
+  margin: 1rem 0 1rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.modal-header .title {
+  font-size: 1.25rem;
+  font-weight: bold;
+}
+
+.modal-header .close-button {
+  cursor: pointer;
+  border: none;
+  outline: none;
+  background: none;
+  font-size: 1.25rem;
+  font-weight: bold;
+}
+
+.modal-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+#overlay {
+  position: fixed;
+  opacity: 0;
+  transition: 200ms ease-in-out;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.9);
+  pointer-events: none;
+}
+
+.modal-img {
+  display: block;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.btn-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.modal-btn {
+  background-color: var(--purple-color);
+  display: flex;
+  justify-content: center;
+  margin: 1.5rem 1rem;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1rem;
+  color: var(--white-color);
+  padding: 0.8rem;
+  transition: all 500ms ease-in;
+  cursor: pointer;
+  text-align: center;
+}
```

Comment:
- [ ] Your live link button seems to have a different Icon compared to the Figma template, kindly change it to conform to the Figma template
- [ ] Your Modal for the mobile version has visual bugs around 390 pixels which is the ideal size for most Mobile devices, kindly make sure your modal does not overflow in the mobile view. You can refer to the images below
- [ ] The margins between the techs and the description are smaller compared to the Figma template, kindly modify it to conform to the Figma template.
> Yours
![image](https://user-images.githubusercontent.com/90556221/167740734-fd4bb2e2-247c-43b2-884e-5d0751908fe6.png)

> Figma
![image](https://user-images.githubusercontent.com/90556221/167740773-2c39dc73-0a55-4228-865f-0bd434340229.png)